src/pages/AddTravel.tsx

```
@@ -72,6 +111,16 @@ const AddTravel = () => {
       </div>
 
       <FloatingMenu onSubmit={handleAddTravel} />
+      <ConfirmModal
+        modalId="zero-price-confirm"
+        trigger={<></>}
+        message={
+          <div css={{ fontWeight: '600', fontSize: '18px' }}>
+            여행 가격이 0원입니다. 계속 하시겠습니까?
+          </div>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)